projects/sunbird-player-sdk/src/lib/core/offline-alert/offline-alert.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input, OnChanges, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'sb-player-offline-alert',
+  templateUrl: './offline-alert.component.html',
+  styleUrls: ['./offline-alert.component.scss']
+})
+export class OfflineAlertComponent implements OnInit, OnChanges {
+  @Input() showOfflineAlert: boolean;
+  constructor() { }
+
+  ngOnInit() {
+
+  }
+
+  ngOnChanges() {
+    if (this.showOfflineAlert) {
+      setTimeout(() => {
```

Comment:
who will send the showOfflineAlert

Reply:
showOffline Alert will be sent by pdf player to sdk , using window.addEventListner